README.md

```diff
@@ -289,12 +239,12 @@ folder to Octave/Matlab `path` variable and call
 If you're not using gdb from the command line, make sure that your IDE is
```

Comment:
```suggestion
If you're not using the debugger from the command line, make sure that your IDE is
```

Reply:
Another legacy section of README. Getting rid of this.